.github/workflows/release-connect-bump-versions.yml

```diff
@@ -0,0 +1,37 @@
+name: "[Release] connect bump versions"
```

Comment:
This will run and create a new unprotected branch with updated versions and create a pull request to the protected branch?

Reply:
This will create a new unprotected branch with update versions and create a pull request to `develop`.
Then we merge it to develop and once it is in develop we run the other workflow [.github/workflows/release-connect-init.yml](https://github.com/trezor/trezor-suite/pull/12444/files/cf2d60fa1b195f90ad1f0602988524c321fab783#diff-5d3b3dd645ad50a3cd17a1ab1ff0c12c48f4c39e0a328c4a13adadfa9602e779) to create the protected branch `release/connect/...`

Reply:
ok